### DISCUSSION TITLE
Use Directus SDK in Angular

### DISCUSSION THREAD

**Discussion by azevedonik** [directus/directus#8600] [Archive]
Hi guys, do you know how to use directus SDK on Angular 10?

I'm getting this error
ng] Error: node_modules/@directus/sdk/dist/base/directus.d.ts:4:10 - error TS2724: Module '"../items"' has no exported member 'IItems'. Did you mean 'Item'?
[ng] 4 import { IItems } from '../items';
[ng]            ~~
[ng] Error: node_modules/@directus/sdk/dist/base/items.d.ts:2:10 - error TS2724: Module '"../items"' has no exported member 'IItems'. Did you mean 'Item'?
[ng] 2 import { IItems, Item, QueryOne, QueryMany, OneItem, ManyItems, PartialItem } from '../items';
[ng]            ~~
[ng] Error: node_modules/@directus/sdk/dist/directus.d.ts:3:10 - error TS2724: Module '"./items"' has no exported member 'IItems'. Did you mean 'Item'?
[ng] 3 import { IItems } from './items';
[ng]            ~~
[ng] Error: node_modules/@directus/sdk/dist/items.d.ts:26:14 - error TS2304: Cannot find name 'Filter'.
[ng] 26     filter?: Filter<T>;

Does anyone have any idea why is this happening?

**Comment by joselcvarela** [comment]
Hello @azevedonik 
I think you are facing the same issue as TomDieNase user reported on Discord: https://discord.com/channels/725371605378924594/819637968259907628/889821556560244738

We found that the version in used was outdated, so my suggestion is to upgrade your typescript to 4.3.5 or newer.
